examples/with-mqtt-js/pages/index.js

```diff
@@ -0,0 +1,71 @@
+import { useState, useRef } from 'react'
+import useMqtt from '../lib/useMqtt'
+
+export const getStaticProps = () => {
+  return {
+    props: {
+      mqttUri: process.env.NEXT_PUBLIC_MQTT_URI,
+      mqttOptions: {
+        username: process.env.NEXT_PUBLIC_MQTT_USERNAME,
+        password: process.env.NEXT_PUBLIC_MQTT_PASSWORD,
+        clientId: process.env.NEXT_PUBLIC_MQTT_CLIENTID,
+      },
+    },
```

Comment:
Not recommended to return keys as props. You can use the keys directly in the component as they are public (start with `NEXT_PUBLIC_`) and can be used anywhere in the app.